.travis.yml

```diff
@@ -15,11 +14,11 @@ matrix:
             apt:
               packages: valgrind
           compiler: gcc
-          env: BUILD_CONFIGURATION=0 WITH_COVERAGE=1
+          env: BUILD_CONFIGURATION=0 WITH_COVERAGE=1 BUILD_DIR="/tmp/cppmicroservices_builds"
```

Comment:
Is the default build path `/tmp/us builds` troublesome because of the space? Even though the `BUILD_DIR` env var is not global now, it is used in all builds except osx builds. So we would be missing CI failures due to spaces in the build path.